helm/ffc-template-node/values.yaml

```diff
@@ -0,0 +1,43 @@
+environment: development
+name: ffc-template-node
+namespace: ffc-demo
+workstream: ffc-demo
+image: ffc-template-node
+
+labels: {}
+
+containerConfigMap:
+  name: ffc-template-node-config-map
+
+deployment:
+  restartPolicy: Always
+  priorityClassName: low
+  replicas: 1
+  minReadySeconds: 5
```

Comment:
Should we default this to 0?  I think we should only delay a pods availability where it is proven necessary, rather than by default.

Reply:
Agreed, done